playbooks/roles/cluster/templates/cluster_provision_survey.json.j2

```diff
@@ -68,6 +80,40 @@
         "variable": "openshift_aws_compute_group_desired_size",
         "question_name": "Compute Nodes",
         "type": "integer"
-      }   
+      },
+      {
+        "question_description": "OPTIONAL Cluster domain if different from default",
+        "min": 0,
+        "default": "",
+        "max": 16,
+        "required": false,
+        "choices": "",
+        "variable": "dns_domain",
+        "question_name": "Domain",
+        "type": "text"
+      },
+      {
+        "question_description": "OPTIONAL AWS access key for DNS access",
+        "min": 0,
+        "default": "",
+        "max": 16,
+        "required": false,
+        "choices": "",
+        "variable": "dns_access_key",
+        "question_name": "DNS Access Key",
+        "type": "text"
+      },
+      {
+        "question_description": "OPTIONAL AWS secret access key for DNS access",
```

Comment:
Not an issue with this PR but just noticed that we have 3 different ways of describing an Optional parameter:

- OPTIONAL
- Optional
- [Optional]

Would you mind choosing one of the above and applying it to the following files please?

```
grep -Ri optional *
playbooks/roles/cluster/templates/cluster_provision_survey.json.j2:        "question_description": "OPTIONAL Cluster domain if different from default",
playbooks/roles/cluster/templates/cluster_provision_survey.json.j2:        "question_description": "OPTIONAL AWS access key for DNS access",
playbooks/roles/cluster/templates/cluster_provision_survey.json.j2:        "question_description": "OPTIONAL AWS secret access key for DNS access",
playbooks/roles/integreatly/files/osd_install_survey.json:        "question_description": "Optional - Only needed for Private Clusters. Single OSD Master Node IP",
playbooks/roles/integreatly/files/osd_upgrade_survey.json:        "question_description": "Optional - Only needed for Private Clusters. Single OSD Master Node IP",
playbooks/roles/integreatly/files/osd_cve_rollout_survey.json:        "question_name": "Product Specific Tags [Optional]",
playbooks/roles/integreatly/files/osd_uninstall_survey.json:      "question_description": "Optional - Only needed for Private Clusters. Single OSD Master Node IP",
```